src/Plugin/Field/FieldWidget/InlineEntityFormMultiple.php

```diff
@@ -240,7 +240,6 @@ public function formElement(FieldItemListInterface $items, $delta, array $elemen
     $element['#element_validate'] = array('inline_entity_form_update_row_weights');
     // Add the required element marker & validation.
     if ($element['#required']) {
-      $element['#title'] .= ' ' . _theme('form_required_marker', array('element' => $element));
```

Comment:
You're right, i also test adding $element['#attributes']['class'][] = 'form-required'; but this add * after fieldset. May be this could be fixed with css.